apigentools/commands/push.py

```diff
@@ -33,6 +33,20 @@ def get_push_branch(self, lang_name):
             pass
         return push_branch
 
+    def git_status_empty(self):
+        # I hope that `--porcelain` doesn't mean this is fragile ¯\_(ツ)_/¯
+        status = run_command(["git", "status", "--porcelain"])
+        result = {}
+        for line in status.stdout.splitlines():
+            line = line.strip()
+            if line:
+                k, v = line.split(maxsplit=1)
+                result[k] = v
```

Comment:
should we append to an array instead, ie `result[k].append(v)` ? Otherwise if you have several files, modified, and `.apigentools` is the last one to be listed, it will be as if it was the only one

Reply:
You're right, the logic is not correct here. Will fix.